src/jit/lsra.cpp

```diff
@@ -373,67 +402,55 @@ regMaskTP LinearScan::stressLimitRegs(RefPosition* refPosition, regMaskTP mask)
 {
     if (getStressLimitRegs() != LSRA_LIMIT_NONE)
     {
+        // refPosition could be null. 
+        // getTempRegForResolution() one such caller.
```

Comment:
Please use a full sentence, e.g. "The RefPosition could be null, for example when called by getTempRegForResolution()."